interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -106,8 +106,22 @@ namespace {
     return false;
   }
 
+  /// \brief Temporarily overrides the current DiagnosticConsumer to supress
```

Comment:
Why `Temporarily`?

Reply:
Fixed (and somewhat extended) documentation.  Thanks!